recipes/dpdk/all/conanfile.py

```diff
@@ -0,0 +1,156 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.env import Environment
+from conan.tools.files import copy, get, export_conandata_patches, apply_conandata_patches, collect_libs, rm, rmdir
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+
+required_conan_version = ">=1.53"
+
+
+class DpdkConan(ConanFile):
+    name = "dpdk"
+    description = "DPDK: a set of libraries and drivers for fast packet processing"
+    # see https://github.com/DPDK/dpdk/blob/main/license/README
+    license = "BSD-3-Clause AND LGPL-2.1-only AND GPL-2.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://dpdk.org"
+    topics = ("networking", "packets", "drivers")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "enable_stdatomic": [True, False],
+        "with_jansson": [True, False],
+        "with_libarchive": [True, False],
+        "with_libbpf": [True, False],
+        "with_libbsd": [True, False],
+        "with_libibverbs": [True, False],
+        "with_libpcap": [True, False],
+        "with_openssl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "enable_stdatomic": False,
+        "with_jansson": True,
+        "with_libarchive": True,
+        "with_libbpf": True,
+        "with_libbsd": False,  # FIXME: libbsd on Conan is outdated?
+        "with_libibverbs": True,
+        "with_libpcap": True,
+        "with_openssl": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.options["libnuma"].shared = True
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libnuma/2.0.19", options={"shared": True})
+        self.requires("libelf/0.8.13")
+        self.requires("zlib/[>=1.2.11 <2]")
+        # self.requires("linux-headers-generic/6.5.9", transitive_headers=True)
+        if self.options.with_jansson:
+            # rte_metrics_telemetry.h
+            self.requires("jansson/2.14", transitive_headers=True, transitive_libs=True)
+        if self.options.with_libarchive:
+            self.requires("libarchive/3.7.6")
+        if self.options.with_libbpf:
+            self.requires("libbpf/1.4.6")
```

Comment:
Hi @valgur!
There is not such version in CCI, could you please check it? 
Thank you!

Reply:
Added a PR for it: #25931. Will revert it for now, though.

Reply:
Oh I see, I'll take a look at that one thanks